app/models/testcase.rb

```diff
@@ -3,6 +3,11 @@ class Testcase < ApplicationRecord
   belongs_to :suite
   has_many :testcase_runs, dependent: :destroy
 
+  validates_presence_of :project_id
+  validates_presence_of :suite_id
```

Comment:
for our cases, i think we almost always have a suite, but not sure that ocato does. we can always create a null_suite